CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 #### Fixes
 
 * Your contribution here.
+* [#1916](https://github.com/ruby-grape/grape/pull/1916): Drop old appraisals - [@NikolayRys](https://github.com/NikolayRys).
```

Comment:
Nitpick: be explicit? How about "drop Rails 3 and 4 support"? If we're no longer testing, we aren't supporting.

Reply:
No, probs, I reword it, but initially my assumption was that you don't support for a long time(since you have removed them from the travis build), so what I did was just a clean-up.

Reply:
Oh I see. I'm good with this.